server/middleware/team.js

```diff
@@ -17,6 +17,22 @@ async function getTeam(req, res, next) {
     }
 }
 
+async function getUnitForTeam(req, res, next) {
+
+    const logger = getLogger(req.request);
+    const { teamId } = req.params;
+
+    try {
+        const response = await infoService.get(`/team/${teamId}/unit`, {}, { headers: User.createHeaders(req.user) });
```

Comment:
This line's too long; consider adding some carriage returns.

Reply:
I have addressed this